### ISSUE TITLE
Downgrade shopify-buy to v2.6.0

### ISSUE THREAD

**Issue by deanbarrow** [AlexanderProd/gatsby-shopify-starter#10]
The cart process fails with shopify-buy v2.6.1 as there appear to be breaking changes.
This is more a workaround than a fix but you need to use version v2.6.0 instead.

See https://github.com/Shopify/js-buy-sdk/issues/687 for more context.

**Comment by AlexanderProd** [comment]
Thanks for letting me know! Can you tell me how to reproduce the bug though since everything seems to work fine for me.

**Comment by deanbarrow** [comment]
It actually relates to another issue here (https://github.com/AlexanderProd/gatsby-shopify-starter/issues/7).
 
The only way to get it to work was to run:

```
gatsby new my-shopify-store https://github.com/AlexanderProd/gatsby-shopify-starter
cd my-shopify-store/
yarn install
yarn upgrade
```
It was the upgrade which then lead to: shopify-buy@2.6.1.

When I set it to 2.6.0 in package.json and ran it again, everything worked as expected.

**Comment by AlexanderProd** [comment]
should be fixed by upgrading  shopify-buy to 2.7.2
